_transactions.md

```diff
@@ -339,6 +339,132 @@ url        | The URL of the 3DSecure confirmation request.
 
 A webpage for 3DSecure confirmation for the user to interact with.
 
+## Beneficiary Information
+
```

Comment:
We should add a small paragraph here, explaining why we sometimes need beneficiary info.